apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNote.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react'
+import django from 'django'
+
+export default class ModerationNote extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      note: '',
+      noteCharCount: 0
```

Comment:
is this needed to be stored in state?
would `this.state.note.length` work wherever you need the charcount?

Reply:
I don't know I just copied the comment but I feel like it was something to do with .length not being a string?! but will try that when we come back to it, thank you!